spec/support/acceptance/puppetserver.rb

```diff
@@ -11,6 +11,6 @@ def unsupported_puppetserver
 
 # These versions only have a single version (x.y.z) released so no upgrade is possible
```

Comment:
This is now outdated

Reply:
You want me to remove the method completely?

Reply:
I think just the comment because when puppetserver is (hopefully) eventually published for Debian 12 then we also can't upgrade until there are at least 2 packages published.

Reply:
You mean to delete the comment, instead of reading it and the next one as "these versions only have a single …: currently none"

Can do